### DISCUSSION TITLE
Glasses glb/gltf model works or not

### DISCUSSION THREAD

**Discussion by nitinrhr183** [jeeliz/jeelizFaceFilter#237] [Q&A] [answered]
I was wondering whether glb model of a glass that I have can be used for rendering the glass in the GlassesVTO demo where you have given the separate json files for lens mesh and frame mesh....Is it possible to use a single whole glb file in the same code?..Thanks in advance

**Comment by xavierjs** [comment, marked as answer]
This demo has been done using the deprecated legacy THREE.js JSON file format.
It would be better now to use a single GLB file indeed.
